pyqtorch/api.py

```diff
@@ -92,11 +92,12 @@ def expectation(
         return inner_prod(state, observable.run(state, values)).real
     elif diff_mode == DiffMode.ADJOINT:
         return AdjointExpectation.apply(
-            circuit, observable, state, values.keys(), *values.values()
+            circuit, state, observable, values.keys(), *values.values()
```

Comment:
Yeah, is there a reason for this ?

Reply:
We mixed the order at some point, now we made it consistent.